modules/openvino_code/shared/device.ts

```diff
@@ -0,0 +1,25 @@
+import { Features } from './features';
+
+enum DeviceId {
+  CPU = 'CPU',
+  GPU = 'GPU',
+  NPU = 'NPU',
+  }
+
+export enum DeviceName {
+  CPU = 'CPU',
+  GPU = 'GPU',
+  NPU = 'NPU',
+  } 
+
+export const DEVICE_NAME_TO_ID_MAP: Record<DeviceName, DeviceId> = {
+  [DeviceName.CPU]: DeviceId.CPU,
+  [DeviceName.GPU]: DeviceId.GPU,
+  [DeviceName.NPU]: DeviceId.NPU,
+};
+
+export const DEVICE_SUPPORTED_FEATURES: Record<DeviceName, Features[]> = {
+  [DeviceName.CPU]: [Features.CODE_COMPLETION],
```

Comment:
Why does the CPU support only code completion?

Reply:
Added all the supported features on CPU